packages/zora/src/Zora.ts

```diff
@@ -3,15 +3,25 @@ import {
   type MintActionParams,
   compressJson,
 } from '@rabbitholegg/questdk'
+import { zoraUniversalMinterAddress } from '@zoralabs/universal-minter'
 import { type Address } from 'viem'
 import { CHAIN_ID_ARRAY } from './chain-ids'
 import { ZORA_MINTER_ABI } from './abi'
+import type { Chains } from './utils'
 
 export const mint = async (
   mint: MintActionParams,
 ): Promise<TransactionFilter> => {
   const { chainId, contractAddress, tokenId, amount, recipient } = mint
 
+  const universalMinter = zoraUniversalMinterAddress[
+    chainId as Chains
+  ].toLowerCase() as Address
+
+  if (!universalMinter) {
+    throw new Error(`no universal minter contract found for ${chainId}`)
```

Comment:
I think it's better to just fail-safe in this case. Can we do a ternary and only pass the array if the `universalMinter` is defined?